Gemfile

```diff
@@ -117,4 +122,11 @@ group :test do
   # gem "capybara", "3.38.0"
   # gem "selenium-webdriver", "4.7.1"
   # gem "webdrivers", "5.2.0"
+  gem 'database_cleaner-active_record'
```

Comment:
all these gems are great, but should be installed as a single PR.
No need to mix so much changes at the same PR